notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxViewModel.kt

```diff
@@ -94,15 +95,15 @@ class NotificationsInboxViewModel(
                 _uiState.value = if (notifications.values.any { it.isNotEmpty() }) {
                     InboxUIState.Data(notifications = notifications.toMap())
                 } else {
-                    InboxUIState.Empty
+                    InboxUIState.Fallback(state = InboxFullScreenState.Empty)
                 }
             } catch (e: Exception) {
                 if (uiState.value is InboxUIState.Data || _isRefreshing.value) {
                     emitErrorMessage(e)
                 } else if (e.isInternetError()) {
-                    _uiState.value = InboxUIState.NetworkError
+                    _uiState.value = InboxUIState.Fallback(state = FullScreenState.NetworkError)
```

Comment:
IMO, better to use `InboxFullScreenState.NetworkError` & `InboxFullScreenState.ServerError`